### PR TITLE
Nexus: Use simplices in convex_hull

### LINKED ISSUE #4670: Nexus delaunay vertices failures due to scipy deprecations
**Describe the bug**
https://cdash.qmcpack.org/CDash/testDetails.php?test=27069120&build=426755

ntest_nexus_numerics is failing with the errors below on multiple machines. The machines have py-scipy@1.11.1

I see via https://docs.scipy.org/doc/scipy/release/1.11.0-notes.html#expired-deprecations that `The vertices keyword of scipy.spatial.Delauney.qhull has been removed (use simplices).`

```
Test output

Test name     : numerics
Test sublabel : test_convex_hull
Test exception: "AttributeError: 'Delaunay' object has no attribute 'vertices'"
Test backtrace:
  File "/scratch/pk7/QMCPACK_CI_BUILDS/qmcpack/nexus/bin/nxs-test", line 478, in run
    self.operation()
  File "/scratch/pk7/QMCPACK_CI_BUILDS/qmcpack/nexus/bin/nxs-test", line 726, in numerics
    nunit_all()
  File "/scratch/pk7/QMCPACK_CI_BUILDS/qmcpack/nexus/bin/nxs-test", line 362, in nunit_all
    run_external_unit_test(test_name,unit_test)
  File "/scratch/pk7/QMCPACK_CI_BUILDS/qmcpack/nexus/bin/nxs-test", line 388, in run_external_unit_test
    unit_test()
  File "/scratch/pk7/QMCPACK_CI_BUILDS/qmcpack/nexus/tests/unit/test_numerics.py", line 294, in test_convex_hull
    hull = convex_hull(points)
  File "/scratch/pk7/QMCPACK_CI_BUILDS/qmcpack/nexus/lib/numerics.py", line 1389, in convex_hull
    v = tri.vertices[ni]

Test status: fail
```
**To Reproduce**

Run develop nexus tests with scipy >=1.11

**Expected behavior**

Tests pass

### PR DESCRIPTION
**Pull request by prckent**
## Proposed changes

Closes #4670 

This looks to be a trivial fix ; vertices was removed in scipy 1.11.0 https://docs.scipy.org/doc/scipy/release/1.11.0-notes.html#expired-deprecations . However I did not yet find the version when simplices was introduced - is it old enough?

## What type(s) of changes does this code introduce?

- Bugfix

### Does this introduce a breaking change?

- No

## What systems has this change been tested on?

sulfur, nightly config

## Checklist

- Yes. This PR is up to date with current the current state of 'develop'

### CODE REVIEW

**Comment by prckent** [comment]
Test this please

**Review by jtkrogel — CHANGES_REQUESTED** [review]

File: nexus/lib/numerics.py [hunk 1]
```diff
@@ -1386,7 +1386,7 @@ def convex_hull(points,dimension=None,tol=None):
             i = ns.index(-1)
             inds = all_inds.copy()
             inds[i] = False
-            v = tri.vertices[ni]
+            v = tri.simplices[ni]
```
[the remark this answers is not in the document]
Comment (prckent) [reply nexus/lib/numerics.py]: Done

**Comment by prckent** [comment]
Test this please

**Comment by prckent** [comment]
Test this please

**Comment by markdewing** [comment]
It looks like "simplices" was added in scipy 0.12 (released in 2013)

**Review by markdewing — APPROVED** [review]

**Comment by markdewing** [comment]
test this please

**Pull request merged**